janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -111,14 +111,14 @@
 
     private static final int DEFAULT_PORT = 9042;
 
+    private static Cluster cluster;
+    private static Map<String,Session> sessions = new ConcurrentHashMap<>();
```

Comment:
We are not running tests in parallel anymore as per the xml file. why does this need to be concurrent? As on line 117 there is only one keyspace variable for this store manager so what else will go in the sessions map? I searched the original file and this file and I do not see any other uses of the sessions map. Finally, initializeSession is private and it seems to be the only method that mutates the session map.